modules/oe_content_sub_entity/oe_content_sub_entity.permissions.yml

```diff
@@ -0,0 +1,7 @@
+administer sub entity types:
+  title: 'Manage content sub entity types'
+  description: 'Allow to administer Sub entity types and their fields'
+
+view unpublished sub entities:
```

Comment:
Not sure that we should manage visibility of sub entities by bundle. We can, but it will additional complexity. 
I think it should be enough - paragraph uses the same approach.